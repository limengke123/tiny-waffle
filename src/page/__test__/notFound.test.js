import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { mount } from 'enzyme'
import NotFound from '../notFound'

describe('notFound', () => {
    it('should render correctly', () => {
        const wrapper = mount(
            <Router>
                <NotFound />
            </Router>
        )
        expect(wrapper.render()).toMatchSnapshot()
    })
})
