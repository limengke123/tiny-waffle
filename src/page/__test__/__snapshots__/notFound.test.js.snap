// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notFound should render correctly 1`] = `
<div>
  we are lost there and find nothing, click
  <a
    href="/counter"
  >
     here 
  </a>
  back to home page !
</div>
`;
